core/src/integrationTest/kotlin/ch/ergon/dope/helper/DBProvider.kt

```diff
@@ -0,0 +1,68 @@
+package ch.ergon.dope.helper
+
+import com.couchbase.client.kotlin.Cluster
+import kotlinx.coroutines.runBlocking
+import org.testcontainers.couchbase.BucketDefinition
+import org.testcontainers.couchbase.CouchbaseContainer
+import org.testcontainers.utility.DockerImageName
+import kotlin.time.Duration.Companion.seconds
+
+object DBProvider {
+    private val container = CouchbaseContainer(
+        DockerImageName.parse("couchbase/server:community-7.6.2"),
+    )
+    var cluster: Cluster
+
+    init {
+        container.withBucket(BucketDefinition("testBucket"))
+        container.start()
+        cluster = Cluster.connect(
+            container.connectionString,
+            container.username,
+            container.password,
+        )
+        fillDatabase()
+    }
+
+    private fun fillDatabase() {
+        val collection = cluster.bucket("testBucket").defaultCollection()
+        runBlocking {
+            repeat(10) {
+                cluster.waitUntilReady(15.seconds)
+                for (i in 1..5) {
+                    collection.upsert(
+                        id = "employee:$i",
```

Comment:
There is a bug: We insert the _employee:1_ 10 times (_repeat(10)_). If you want to have 50 employees in the database afterwards you should change the id

Reply:
I purposefully only add 5 of each type, but as discussed changed it to use the tryUntil